src/torcontrol.cpp

```diff
@@ -608,7 +608,9 @@ void TorController::disconnected_cb(TorControlConnection& conn)
     service = CService();
     if (!reconnect)
         return;
-    LogPrintf("tor: Disconnected from Tor control port %s, trying to reconnect\n", target);
+
+    LogPrint("tor", "tor: Disconnected from Tor control port %s, trying to reconnect\n", target);
```

Comment:
@dcousens Yes, because the "if there was a connection in the first place" logic is not here yet.